iceoryx_examples/icedelivery_in_c/topic_data.h

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef IOX_EXAMPLES_ICEDELIVERY_ON_C_TOPIC_DATA_HPP
-#define IOX_EXAMPLES_ICEDELIVERY_ON_C_TOPIC_DATA_HPP
+#ifndef IOX_EXAMPLES_ICEDELIVERY_in_c_TOPIC_DATA_HPP
```

Comment:
How I love `#pragma once` and hate these macro guards and on that note: They have to be all CAPITAL_SNAKE_CASE and you forgot `_in_c_`.

Reply:
Thanks for this catch. That's why one should always double check a search and replace...

Reply:
..and use match case. Fixed.